### PR TITLE
Add the full ODbL license

### PR DESCRIPTION
**Pull request by johanricher**
Hi,

The full text of the license needs to be stated clearly to avoid ambiguity. Without this disambiguation, ODbL is not recognized and applied correctly. This will also allow make it fully parsable, for machine-readable purposes.

Thanks for this project and your great work!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,540 @@
...
+
+and 
+
+You (as defined below) 
+
+agree as follows: 
+
+### 1.0 Definitions of Capitalised Words
+
+"Collective Database" â€“ Means this Database in unmodified form as part
```
Comment (mledoze) [inline LICENSE]: What are those `â€“` characters?

**Comment by mledoze** [comment]
Hello @johanricher no problem, thank you for the pull request.

File: LICENSE (hunk 1 above)
Comment (mledoze) [reply LICENSE]: Could you please replace all occurrences of `â€“` with `–`.

**Comment by johanricher** [comment]
Sorry, weird encoding problem on my end with [the original file](https://www.opendatacommons.org/files/2018/02/odbl-10.txt).

File: LICENSE (hunk 1 above)
Comment (johanricher) [reply LICENSE]: Sorry, weird encoding problem on my end with [the original file](https://www.opendatacommons.org/files/2018/02/odbl-10.txt).

Comment (mledoze) [reply LICENSE]: Yes I saw that, thank you.

**Comment by johanricher** [comment]
Merci Mohammed ! Bonne journée.

**Pull request merged**